### PR TITLE
wiki - add 'write' 'preview' buttons to wiki edit like in issues

### PR DESCRIPTION
**Pull request by Cherrg**
To unify wiki and issues:

this will add 'write' and 'preview' buttons above wiki

affects #6975

## Image

grafik

**Dark Theme**
grafik

(Note: hide whitespaces changes to see added code)

Signed-off-by: Michael Gnehr <michael@gnehr.de>

### CODE REVIEW

**Comment by silverwind** [comment]
I think you can remove the "preview" button in the toolbar, right?

**Comment by Cherrg** [comment]
i kept the button to:
 * to allow toggle on full screen
 * keep user experience -> so people who used this button still find it in place

I could change the behavior to clean up the button bar (too much symbols...) and only show it when codemirror is in fullscreen mode

**Comment by Cherrg** [comment]
done -> only show preview button on fullscreen

**Comment by lunny** [comment]
It seems it's not work.

**Comment by Cherrg** [comment]
resolved conflicts, should work now

**Comment by lunny** [comment]
@Cherrg resolved again. :)

**Comment by Cherrg** [comment]
resolved :)

**Review by 6543 — APPROVED** [review]

**Comment by Cherrg** [comment]
don't know why drone build failed as this does not change anything in searchIndexer or mysql8. May someone restart drone build. Thy

**Comment by silverwind** [comment]
You can restart yourself with a commit: `git commit --allow-empty -m "restart ci"`

**Comment by techknowlogick** [comment]
I've just restarted the build

**Comment by Cherrg** [comment]
It was just too easy. But thy :)

File: web_src/js/index.js [hunk 1]
```diff
@@ -1232,6 +1232,46 @@ function initWikiForm() {
         'clean-block', 'preview', 'fullscreen', 'side-by-side']
     });
     $(simplemde.codemirror.getInputField()).addClass('js-quick-submit');
+
+    setTimeout(() => {
```
Comment (lunny) [inline web_src/js/index.js:1236]: How about to create a wiki.js and move PR related functions to that file?

Comment (Cherrg) [reply web_src/js/index.js:1236]: should this still be compiled into the same lines?
(as this need to be loaded after simpleMDE is ready)

Comment (Cherrg) [reply web_src/js/index.js:1236]: line 1272 `sideBySideChanges = 10;` changes variable from line 1137 `let sideBySideChanges = 0;`

Comment (Cherrg) [reply web_src/js/index.js:1236]: when all simpeMDE related stuff is in there (line 1146 has FIXME and will be fixed in #7390 or #8891), i will add all wiki related simpleMDE changes to simpleMDE element near line 1304.
May i place the simpleMDE stuff into an extra file?

Comment (silverwind) [reply web_src/js/index.js:1236]: We could certainly start lazy-loading simpleMDE if it's only used in a few places, ideally using it directly from npm as a dependency.

**Comment by lunny** [comment]
@Cherrg LGTM except I think js file should be `wiki.js`.

**Pull request merged**